samples/snippets/src/main/java/com/example/spanner/SpannerSample.java

```diff
@@ -1258,6 +1276,7 @@ static void createTableWithDatatypes(DatabaseAdminClient dbAdminClient, Database
                     + "  OutdoorVenue    BOOL, "
                     + "  PopularityScore FLOAT64, "
                     + "  Revenue         NUMERIC, "
+                    + "  VenueDetails  JSON, "
```

Comment:
nit: try to align with the line above so the data type starts at the same column